### ISSUE TITLE
(gift giving) vs (funding) vs legal weapons of FUD

### ISSUE THREAD

**Issue by techtonik** [gratipay/inside.gratipay.com#201]
Previously Gratipay was gratitude gift giving platform, but.. we evolved into something called "Teams", which is essentially a way to backup more complicated social systems that stand behind software products, and by doing this we stepped up on some legal labor and regulation minefield. So we had to crop all the fat and concentrate only on one thing.

Still. we are leaving a lot of people who liked to give and receive gifts behind without any human explanation. That's bad.. I think we need a visual clarity (illustration or drawing) of this legal problem. Due to legal FUD I see that Gratipay fails to communicate what going on, fails to preserve openness. I don't think that this is a specifically designed attack. Rather that's a flashback mechanism of old school legal warfare system.

I found that many countries have their laws that govern gift giving. These laws are separate from taxation laws, and there is still a demand to have a platform that clarifies and dispels the FUD that this is criminal, money laundering or something.

I live in Belarus and I ..ing know how is to be afraid of everything legal. I am tired.

**Comment by mattbk** [comment]
+1

**Comment by clone1018** [comment]
+1

**Comment by colindean** [comment]
It's all messed up because, basically, government has an interest in ensuring that all transfers between people are tracked, for taxation purposes primarily and monitoring for criminal activity secondarily. Gratipay attempted to exist in a reality where everyone was honest: receivers self-report their income and donors self-report gifts as necessary to the necessary taxing authorities, and donors are donating only their own money to receivers who aren't doing illegal things with the money. Unfortunately, this reality *isn't* real because of dishonest people outside of Gratipay's conceptualization and worldview.

In short, criminals are why we honest people can't have nice things, like anonymous generosity through no-strings-attached gifts.

</rant>

**Comment by techtonik** [comment]
If all the problem is anonymity, we can disable it, so that people could see who gives how much to who. But we COULD do better just by explaining to people that we need to comply, or else we will be punished (what is the punishment, BTW?). Therefore we *have* to provide the data to U.S. government, but we can:

1. provide data only on transactions with US citizens involved
2. track access to that data from goverment channels (in case they are hacked)
3. still provide anonymity for general public
4. explain what do you need to do to stay legal (which taxes you should pay, what paper to fill)
5. fill and submit stuff from (4) automatically in electronic form

**Comment by colindean** [comment]
You've got it at a high level. But big companies with budgets and compliance departments mess this up. I'm not saying Gratipay *shouldn't* try, but rather that the burden of regulation makes it very difficult to comply on a bootstrapped startup's budget.

**Comment by whit537** [comment]
@techtonik Colin is basically right. The U.S. government is primarily concerned with taxes and terrorism. State governments also come into play, and they're mostly concerned with making sure consumers don't get ripped off.

> [W]hat is the punishment, BTW?

In the U.S., our federal law is codified in the [United States Code](http://www.gpo.gov/fdsys/browse/collectionUScode.action?selectedYearFrom=2010) (U.S.C.) which is divided into 53 "Titles." Here's the sort of taxonomy one finds there:

```
United States Code
  Title
    Part (or Subtitle?)
      Chapter
        (Subchapter?)
          (Part, again?)
            Section
              (a)
                (1)
                  (A)
                    etc.
```

Generally, references are to "[Title] U.S.C. [Section]," so, for example, [18 U.S.C. §1960](http://www.gpo.gov/fdsys/pkg/USCODE-2010-title18/html/USCODE-2010-title18-partI-chap95-sec1960.htm) is where we find the United States' "Prohibition of unlicensed money transmitting businesses":

> Whoever knowingly conducts, controls, manages, supervises, directs, or owns all or part of an unlicensed money transmitting business, shall be fined in accordance with this title or imprisoned not more than 5 years, or both.

Title 18 is criminal law (as opposed to civil law; here's a description of the [difference](http://www.rbs2.com/cc.htm)). [31 U.S.C. §5330](http://www.gpo.gov/fdsys/pkg/USCODE-2010-title31/html/USCODE-2010-title31-subtitleIV-chap53-subchapII-sec5330.htm) is the civil law related to "Registration of money transmitting businesses":

> Any person who fails to comply with any requirement of this section or any regulation prescribed under this section shall be liable to the United States for a civil penalty of $5,000 for each such violation. [...] Each day a violation [...] continues shall constitute a separate violation[.]

Now, there's also the [Code of Federal Regulations](http://www.ecfr.gov/cgi-bin/ECFR) (C.F.R.) which details how various U.S. government agencies carry out the laws recorded in U.S.C. [31 C.F.R. §1010.100](http://www.ecfr.gov/cgi-bin/text-idx?SID=368bfe5df617f990d014eade2b231d08&mc=true&node=pt31.3.1010&rgn=div5#se31.3.1010_1100) gives a bunch of definitions, including this definition of a money transmitter: "A person that provides money transmission services[, or a]ny other person engaged in the transfer of funds." It also provides for exceptions to this definition, and the one we're designing [our new terms of service](https://github.com/gratipay/gratipay.com/pull/3408) to fit is the "payment processor" exemption:

> The term “money transmitter” shall not include a person that only: [...] Acts as a payment processor to facilitate the purchase of, or payment of a bill for, a good or service through a clearance and settlement system by agreement with the creditor or seller;

The :rabbit: hole goes deeper, of course (I haven't touched on [case law](https://casetext.com/case/us-v-elfgeeh) or FinCEN [guidance](http://fincen.gov/statutes_regs/guidance/), let alone [state money transmitter laws](https://www.coinbase.com/legal/licenses)). Hopefully, though, this gives you some sense of the legal concerns with our old terms of service.

And all that said, I really do [believe](https://github.com/gratipay/inside.gratipay.com/issues/180#issuecomment-100497390) that "Legal concerns made us stop with our old model, but it's business vision that is driving our new model." Teams are one of the most interesting and successful things we've discovered. Yes, it's hard to give up distributed genius grants to individuals, but we're giving up one good thing in order to focus on another good thing. The pieces are coming together so that we *may* be able to run a limited payday this Thursday, and I think that once we start processing again we're going to realize that the worst is behind us and we've got a fun and interesting product to develop together.

**Comment by whit537** [comment]
> provide data only on transactions with US citizens involved
> still provide anonymity for general public

Actually, as I understand it, the U.S. government [requires us to report](http://www.ecfr.gov/cgi-bin/text-idx?SID=368bfe5df617f990d014eade2b231d08&mc=true&node=pt31.3.1010&rgn=div5#se31.3.1010_1320) any suspicious and/or large transactions.

> explain what do you need to do to stay legal (which taxes you should pay, what paper to fill)
fill and submit stuff from (4) automatically in electronic form

I agree that this is an area where Gratipay can provide value. I see us as trying to model the legal system in a way that hackers can understand and hopefully cope with. Yes, it's a big ugly (and sometimes scary and depressing) mess, but I think we're actually moving in the right direction by first of all getting our own house in order.

**Comment by techtonik** [comment]
!m @whit537 I took me a day to munch through this and I am not finished yet. =/

**Comment by whit537** [comment]
@techtonik No worries, I had a week to munch on this non-openly with @copiesofcopies' help, and as I say the :rabbit:-hole goes deeper still.

**Comment by whit537** [comment]
I'd like to explain why we can't pay out in bitcoin anymore.

To start, allow me to introduce the Financial Crimes Enforcement Network (FinCEN), a bureau of the U.S. Treasury Department. They are [charged](http://fincen.gov/about_fincen/wwd/) with monitoring the financial system and enforcing **law** related to it, which they do in accordance with the **regulations** in [31 C.F.R. Chapter X](http://www.ecfr.gov/cgi-bin/text-idx?SID=d657edc33d93acba7fbd22e30696bf76&mc=true&tpl=/ecfrbrowse/Title31/31chapterX.tpl) (see [above](https://github.com/gratipay/inside.gratipay.com/issues/201#issuecomment-101516359) for the difference between law and regulations; basically regulations are details on how to implement law). FinCEN [publishes](http://fincen.gov/statutes_regs/guidance/) **guidance** on their application of law and regulations, which includes a selection of **rulings** on specific cases. Any company may write to FinCEN with a "request for ruling." Gratipay may do this at some point. It's unclear how long it takes to hear back, but it's probably longer than a week. ;-)

To be honest, I don't think this is a bad system. The guidance FinCEN provides is by and large clear and specific, and gives us the information we need to feel confident that we're acting lawfully (and in any case where things aren't clear, we can always ask). For example, in [FIN-2013-R002](http://www.fincen.gov/news_room/rp/rulings/pdf/FIN-2013-R002.pdf) they clearly spell out the criteria they use to decide whether a company fits the payment processor exemption to the definition of a money transmitter, to which they then refer in [FIN-2014-R012](http://www.fincen.gov/news_room/rp/rulings/pdf/FIN-2014-R012.pdf) when deciding whether a hotel chain that accepts payments via credit card and pays out via bitcoin fits the exemption:

>FinCEN stipulates four conditions for the payment processor exemption to apply to a particular business pattern:
>
(a) the entity providing the service must facilitate the purchase of goods or services, or the payment of bills for goods or services (other than money transmission itself);
>
(b) the entity must operate through clearance and settlement systems that admit only BSA-regulated financial institutions;
>
(c) the entity must provide the service pursuant to a formal agreement; and
>
(d) the entity’s agreement must be at a minimum with the seller or creditor that provided the goods or services and receives the funds.
>
The Company fails to satisfy one of these conditions. The Company is not operating through clearing and settlement systems that only admit BSA-regulated financial institutions as members. According to your letter the real currency payments from the consumer take place within a clearing and settlement system that only admits BSA-regulated financial institutions as members (specifically, a credit card network), however, the payment of the Bitcoin equivalent to the merchant, by definition, takes place outside such a clearing and settlement system, either to a merchant-owned virtual currency wallet or to a larger virtual currency exchange that admits both financial institution and non-financial institution members, for the account of the merchant.

Clearly, we can't take in money from credit cards and send it out via bitcoin, and qualify for the payment processor exemption of the definition of a money transmitter. Now, for context (as I mentioned [last week](https://github.com/gratipay/inside.gratipay.com/issues/192#issuecomment-100048271)), we've only done 4 bitcoin payouts vs. 516 ACH credits and 468 PayPal payouts since we started keeping better track with gratipay/gratipay.com#3282. However, I thought we should provide an explanation for our few users that had been using bitcoin to withdraw.

**Comment by mattbk** [comment]
Bitcoin still mentioned at https://gratipay.com/about/faq

**Comment by silverhook** [comment]
Scraping Bitcoin (for now) seems as an understandable course of action. It’d be great if someday on-line P2P financing would work, but we’re not there yet and the legislators (through banks) are still too scared of it.

P.S. BSA = Bank Secrecy Act …got me *really* worried for a second there :laughing:

**Comment by whit537** [comment]
@mattbk Good call. Should be an easy PR for someone. :-)

**Comment by whit537** [comment]
@techtonik What are the action items remaining to close this ticket?

**Comment by techtonik** [comment]
@whit537 send a letter when and why we've killed Bitcoin

**Comment by techtonik** [comment]
@whit537 send an announcement why we killed Bitcoin. Am I right that it is because we tried to send stuff between Bitcoin to Credit Card (through Coinbase)? And for doing this we need a license  are doing 

To close this ticket, we need to draw a picture or fund a movie explaining why Gift Giving is illegal, how to make it legal, and where Gratipay is moving.

**Comment by whit537** [comment]
> send a letter when and why we've killed Bitcoin
> send an announcement why we killed Bitcoin.

We've processed payments (payins and/or payouts) for 4,222 users, of whom 29 (0.7%) have used bitcoin, of whom 15 (52%) have an email address on file (though we may have some more email addresses [hiding in `exchange_routes`](https://github.com/gratipay/gratipay.com/issues/3541)). I'd be fine sending an email to these 15 people. Does that sound right, @techtonik?

> Am I right that it is because we tried to send stuff between Bitcoin to Credit Card (through Coinbase)? 

Yes, more or less. As discussed above at https://github.com/gratipay/inside.gratipay.com/issues/201#issuecomment-102010517, the precise reason is that one of the four requirements to qualify for the payment processor limitation on the definition of a money transmitter is that we "must operate through clearance and settlement systems that admit only BSA-regulated financial institutions" (BSA is the Bank Secrecy Act, the U.S.A.'s anti-money laundering law, as discussed at https://github.com/gratipay/inside.gratipay.com/issues/119#issuecomment-107081615).

> And for doing this we need a costly license of money transmitter. What was the last price, BTW?

Getting licensed as a money transmitter means going through a lengthy and costly approval process ... 48 times. Because it's such a complex process, there is no fixed price; the order-of-magnitude estimate is "millions." It's the cost of producing web pages such as these:

https://www.paypal.com/webapps/mpp/licenses
https://payments.amazon.com/help?nodeId=6026
https://www.coinbase.com/legal/licenses

**Comment by whit537** [comment]
>To close this ticket, we need to draw a picture or fund a movie explaining why Gift Giving is illegal, how to make it legal, and where Gratipay is moving.

Okay! :-)

Now that we've discovered blockdiag (https://github.com/gratipay/inside.gratipay.com/issues/247 !m @techtonik :-), below is a visual representation of the prose at https://github.com/gratipay/inside.gratipay.com/issues/192#issuecomment-100048271 (which is, incidentally, nearly the first comment I made after emerging from private talks with our lawyer). Refer to that comment for further explanation of the diagram; the two (prose and diagram) go together. I've split the diagram into three parts to work around limitations with the blockdiag interactive editor (I stalled out installing blockdiag locally). Red means regulated, green means not *directly* regulated (cf. https://github.com/gratipay/inside.gratipay.com/issues/119#issuecomment-107081615). Images link to interactive blockdiag.

[![regulated](https://cloud.githubusercontent.com/assets/134455/8109640/d426666e-1025-11e5-8a9a-ef5a2d2a82af.png)](http://interactive.blockdiag.com/?compression=deflate&src=eJxtUz1v3DAM3f0rhOvQpc2eIV0CZC4QZKo7yBLlU22TAiUdLijuv5eybPkaZHuUnvj48TTMZCbr9aj-dmpkykExjHnWCWw5MjQTPzHYrlPOq1-zHmB-OjmPGo3Xc48eY_IpJ094-rbxT8HjdPotbwaNU3tVAjlVEcxxxjQBK2JlQc_AJWEhZPbJQyz0JQ6NvhDCu9zzxRuIPQ45eoQYP5F21_aqplYee3TE4EdUcDVnjSMUAXMGM8XGXkNldDwDr_rUrnyMuVYbYV6rJZdYX0Dw17glkkNWtVBiC7w2EUJoWQLTxduSx_UYGIL2VmljSh9FL31oVxQwLj4lqedjnyrHcJT-9vD6oH5STHpWr3VIVfyg1MI_l-5UGhsxSVMj63AWxSVofF9HpWXexzxquO403_UnPZNsMQ9_wKQeE5XrUk4sLhG2y0ePgjNDLCqLzFcYPS7AZT1plaTlmD8mJpuNx1FV56x-KU_JHoZxxwAluczCZbTVFF59_7G6csfitR2K0XaYxh3VFhs7hx1K3Y1DS8vhOlUtWyjXBqs1WrhQg-KM4zg1WPZ6xxF868S1sITy1WSfmw--PD6-vDw_b8YR7sHZf3TG__705qCRAVCie_6tu_0DKVxeqA)

[![exemptions](https://cloud.githubusercontent.com/assets/134455/8109253/ad8fe900-1023-11e5-8655-08ead7de8666.png)](http://interactive.blockdiag.com/?compression=deflate&src=eJxtUsFuwjAMvfMVEbts0sadA7sg8RPrNLmpKRapEyUuWzXx73NaaBnlFr_YL-89p3TeHiuC2vwuTB19G0zEunUgWGXIWO983ESsFlrsyXw4KNFtlntiYEvgCiZOQtIKeV6-XgaWgfi4_MxDXzAOrVYrxYxpUjlijWfsTMJ4Ioup4LJNxJjSjEqZyjumjNk5u9yRSwRODYlgfMRazVnV6Nu7Kh-PqniC9ajFYCPX9rZq5N9dzu2s7fiDTcgRpfG1CSr4GY6gPRYH4LVgR6oY-uplUv20Xu92220Wrjb1gYnkur-WH22wjoicrfG0Dkb59vFowGryKuK6BBOiP1GlaeV8QhgHAnQNsuT7POHjQ2HGTh6tQ9D0LRrgKr8g4rDnSF0SbIYXDt00kQuy4FxXcL-44KMkY9sYkW13kQQ1EE8zEQNQdTHStxAL1hHc2DMC4o0csODeZNLkjN-P_29Y_82yNGKGGRTCDLJp3qVWHoz22mf4VZ9-l8X5D7nQLYQ)

[![conditions](https://cloud.githubusercontent.com/assets/134455/8109401/75ba6504-1024-11e5-991f-b2cbb781cbc2.png)](http://interactive.blockdiag.com/?compression=deflate&src=eJxtksFOwzAMhu99CqtcQBq77zAkQNoLcCRo8lK3i5baUZICE9q7k6wjnVhv8a_ffz472VnRh8ZgBz8VdF4GB566wWKkJkugxYpfe2qqVPQR3i3uyK7rXpiOED1y6E2M5OvFxVs7w4f6I_lPUAF9U--iEQ6ldZIU3-MBk0fTKCwUW5Py8Fw9TJl3q9Vm8_qaYjPE49NV7h_3wHPknSfizL6lArBcLnMQOFckh8eeOILzoikE8bNXw7b9F5K0qwkT15ZuJOduXW3udIPXewwTWIva2Dw9QdyT4mKQVnEn0gQQD4H8p0mU84gvb88lT9gep_dU3BpG1gatYsMhmjicieaDWvE92oltLDEvNK9qvimQteRLU3GDCYARLGGIir9M3OcJL_a5qPF58vYuOyhCGrCcR6hSjnH551WnX7LD7k8)

**Comment by whit537** [comment]
How's that for drawing a picture, @techtonik? Satisfied?

If so, the last action item on this ticket is emailing the 15 contactable users who used bitcoin, yes?

**Comment by whit537** [comment]
> If so, the last action item on this ticket is emailing the 15 contactable users who used bitcoin, yes?

I think we missed the window of timeliness on this. :-(

Closing.
